aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,536 @@
+            }
+            },
+            "deprecated": false
+        }
+        },
+        "/api/Todo/{id}": {
+        "get": {
+            "tags": [
+            "Todo"
+            ],
+            "operationId": "ApiTodoByIdGet",
+            "consumes": [],
+            "produces": [],
+            "parameters": [
+            {
+                "name": "id",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            }
+            ],
+            "responses": {
+            "204": {
+                "description": "No Content"
+            }
+            },
+            "deprecated": false
+        },
+        "put": {
+            "tags": [
+            "Todo"
+            ],
+            "operationId": "ApiTodoByIdPut",
+            "consumes": [
+            "application/json",
+            "text/json",
+            "application/json-patch+json"
+            ],
+            "produces": [],
+            "parameters": [
+            {
+                "name": "id",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            },
+            {
+                "name": "item",
+                "in": "body",
+                "required": false,
+                "schema": {
+                "$ref": "#/definitions/TodoItem"
+                }
+            }
+            ],
+            "responses": {
+            "204": {
+                "description": "No Content"
+            }
+            },
+            "deprecated": false
+        },
+        "delete": {
+            "tags": [
+            "Todo"
+            ],
+            "operationId": "ApiTodoByIdDelete",
+            "consumes": [],
+            "produces": [],
+            "parameters": [
+            {
+                "name": "id",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            }
+            ],
+            "responses": {
+            "204": {
+                "description": "No Content"
+            }
+            },
+            "deprecated": false
+        }
+        }
+    },
+    "definitions": {
+        "TodoItem": {
+        "type": "object",
+        "properties": {
+            "key": {
+            "type": "string"
+            },
+            "name": {
+            "type": "string"
+            },
+            "isComplete": {
+            "type": "boolean"
+            }
+        }
+        }
+    },
+    "securityDefinitions": {}
+    }
+
+This document is used to drive the Swagger UI which can be viewed by navigating to ``http://localhost:<random_port>/swagger/ui``
+
+.. image:: web-api-help-pages-using-swagger/_static/swagger-ui.png
+
+Each of the methods in the ``ToDo`` controller can be tested using the UI be clicking the method to expand the section, completing the parameters (if required) and clicking the "Try it out!" button to see the result.
```

Comment:
Each of the methods in the ``ToDo`` controller can be tested from the UI. <avoid run-on sentences> Tap a method to expand the section, add any necessary parameters and tap "Try it out!".